docs/Upgrading-2.0-to-3.0.md

```diff
@@ -275,9 +278,9 @@ The pallet has been moved to a new system in which the exact amount of deposit f
  	type DesiredMembers = DesiredMembers;
  	type DesiredRunnersUp = DesiredRunnersUp;
  	type TermDuration = TermDuration;
- ```
+```
 
- **This upgrade requires storage [migration](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/migrations_3_0_0.rs)**. Further details can be found in the [pallet-specific changelog](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/CHANGELOG.md#security).
+**This upgrade requires storage [migration](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/src/migrations_3_0_0.rs)**. Further details can be found in the [pallet-specific changelog](https://github.com/paritytech/substrate/blob/master/frame/elections-phragmen/CHANGELOG.md#security).
```

Comment:
Do you have a formatter running in your IDE or why the unrelated changes in the markdown files?

Reply:
Its the formatter... I will undo and commit